README.md

```diff
@@ -1,32 +1,43 @@
-## Grapl
-
-Grapl is a Graph Platform for Detection and Response with a focus on
-helping Detection Engineers and Incident Responders stop fighting
-their data and start connecting it. Grapl leverages graph data
-structures at its core to ensure that you can query and connect your
-data efficiently, model complex attacker behaviors for detection, and
-easily expand suspicious behaviors to encompass the full scope of an
+<p align="center">
```

Comment:
suggestion: hit the "display rich diff" button in the top right next to 'viewed'